src/components/Purchase/CheckoutMessageCard.js

```diff
@@ -59,7 +59,7 @@ const CheckoutMessageCard = ({
                     totalItemsPurchased === 1 ? 'album' : 'albums'
                   } has been completed.`}
                   <br />
-                  {`A confirmation email was sent to ${userEmail}`}
+                  {/* {`A confirmation email was sent to ${userEmail}`} */}
```

Comment:
I think how this was before was correct; 

`user` from `useAuth` returns a nested object like:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/67df8b11-0579-409a-9f12-35c6f94bd768)

so we do have to access the email by going `user.user.email`